frontend/static/quotes/english.json

```
@@ -38794,6 +38794,12 @@
       "source": "Fred Rogers",
       "length": 213,
       "id": 6815
+    }
+	{
+      "text": "I'm sorry. I'm sorry. I'm sorry. It's all to save Eldia... and the Island. But not only because of that. The reality outside the walls... was nothing like the world I had dreamed of. It was nothing like the world I had seen in Armin's book. When I learned that humanity outside the walls survived... I... was so dissapointed. I... wished... for everything to be wiped away.",
```

Disappointed is misspelled.
```suggestion
      "text": "I'm sorry. I'm sorry. I'm sorry. It's all to save Eldia... and the Island. But not only because of that. The reality outside the walls... was nothing like the world I had dreamed of. It was nothing like the world I had seen in Armin's book. When I learned that humanity outside the walls survived... I... was so disappointed. I... wished... for everything to be wiped away.",
```